src/pgo/env.rs

```diff
@@ -32,17 +32,17 @@ pub fn find_pgo_env() -> anyhow::Result<PgoEnv> {
     }
 
     // Try to find `llvm-profdata` directly in PATH
-    if let Ok(llvm_profdata) = resolve_binary(Path::new("llvm-profdata"))
+    match resolve_binary(Path::new("llvm-profdata"))
```

Comment:
You can revert this automated change, the `if let` is fine.

Reply:
Ok, reverting the standard fixes:
https://doc.rust-lang.org/cargo/commands/cargo-fix.html